templates/registration/login.html

```diff
@@ -124,6 +125,7 @@ <h2 data-i18n-key="outside-pages/login-header-title">Sign in to <span id="app-na
                             </div>
                             <p class="login-links">
                                 <a href="/accounts/password/reset/" id="password-recovery-link" data-i18n-key="forgot-your-password-link">Forgot your password?</a>
+                                <span> <a href="https://ncds.ona.io/join/">Need an account?</a></span>
```

Comment:
Should this be displayed conditionally?

Reply:
In django templating, it would be something of the sort
```
# get join link as boolean
{% settings_value "join_link" as add_join_link %}
# use boolean join link to decide if we should add join link or not
{% if add_join_link %}
<span> <a href="{% settings_value 'join_link' %}/">Need an account?</a></span>
{% endif %}
    ```

Reply:
Noted...